homeassistant/components/weblink.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+"""
+homeassistant.components.weblink
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Adds links to external webpage
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/webpage/
+"""
+
+# The domain of your component. Should be equal to the name of your component
+import logging
+
+from homeassistant.helpers.entity import Entity
+
+DOMAIN = "weblink"
+
+# List of component names (string) your component depends upon
+DEPENDENCIES = []
+
+ATTR_NAME = 'name'
+ATTR_URL = 'url'
+
+DEFAULT_WEBLINK_ICON = 'mdi:open-in-new'
```

Comment:
Default icons shouldn't be in the component but have to be defined in the frontend.